angular.json

```diff
@@ -879,6 +879,103 @@
                     }
                 }
             }
+        },
+        "ecometrics": {
```

Comment:
There are warnings when starting the app.
You need to define some CommonJS bailouts (cf. other apps), e.g. for codemirror

Reply:
cb465ee97b1974e646abb06d7d40bb98af4611d9